src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -642,7 +642,7 @@ button:focus,
 input:focus,
 select:focus,
 textarea:focus {
-  outline: 2px solid #f9c642;
+  outline: 2px solid $color-gold-light;
```

Comment:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L85